docs/codelabs/building-beautiful-sites/docsite/3-toolbar.md

```diff
@@ -17,44 +17,104 @@ edited for a non-technical audience
-      <a id="shrine-nav-icon" class="material-icons" href="#"
-         aria-label="Click to show the navigation menu"
-         aria-controls="shrine-nav-menu">menu</a>
-      <h1 id="shrine-logo"
+			<a id="shrine-nav-icon" class="material-icons mdc-ripple-surface"  
+			    href="#" 
+			   aria-label="Click to show the navigation menu"
+			   aria-controls="shrine-nav-menu"
+			   data-mdc-auto-init="MDCRipple"
+			   data-mdc-ripple-is-unbounded>menu</a>
+      <h1 id="shrine-logo"          
           class="mdc-toolbar__title"><span>Shrine</span></h1>
     </section>
   </div>
-</header>
+	</header>
+  <main class="mdc-toolbar-fixed-adjust">
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$20</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Sunglasses" src="assets/sunnies.png">
+    </div>
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$80</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Shoes" src="assets/chucks.png">
+    </div>
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$90</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Beach ball" src="assets/beachball.png">
+    </div>
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$30</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Backpack" src="assets/backpack.png">
+    </div>
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$90</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Clock" src="assets/clock.png">
+    </div>
+    <div class="mdc-card shrine-product-card">
+      <section class="mdc-card__primary">
+        <span class="mdc-card__title shrine-product-card__price">$30</span>
+      </section>
+      <img class="shrine-product-card__image" width="240" height="240" alt="Fish bowl" src="assets/fishbowl.png">
+    </div>
+  </main>
+	<script src="https://unpkg.com/material-components-web@0.9.1/dist/material-components-web.min.js">
+	</script>
+	<script>mdc.autoInit()</script>
+</body>
+</html>
 ```
 
-## Add the custom mdc-toolbar styles
+And save.
 
-Add the following styles to `app.css`:
+## Change the styles
+
+Copy and paste the following into `app.css` at the bottom, after 
+```html 
+.shrine-product-card__image {
+  margin: 0 auto;
+}
+```:
```

Comment:
this did not render as expected because of the : after the ```

Might I suggest something like:

    In `app.css`, find this section near the bottom:

    ```html
    .shrine-product-card__image {
      margin: 0 auto;
    }
    ```

    Right below that, copy and paste the following:

Reply:
Done